### PR TITLE
add identity verification page family link

### PR DESCRIPTION
**Pull request by harshared**
ME-180412223

### CODE REVIEW

**Review by markgoho — CHANGES_REQUESTED** [review]
please remove ticket number from PR title and reference in comment box as ME-180412223

**Review by markgoho — DISMISSED** [review]
updated with ticket information, thanks @harshared

File: features/step_definitions/individual_steps.rb [hunk 1]
```diff
@@ -1023,6 +1023,11 @@ class VlpDocument
   expect(page).to have_content "My #{Settings.site.short_name}"
 end
 
+Then(/Individual should land on help paying coverage page$/) do
+  sleep 1
+  expect(page).to have_content "Upload document(s) proving your identity"
```
Comment (markgoho) [inline features/step_definitions/individual_steps.rb:1028]: please use a page object reference or a css class/attribute, but not hard-coded english text

**Review by markgoho — CHANGES_REQUESTED** [review]

File: features/support/object_model_pages/your_information.rb [hunk 2]
```diff
@@ -27,4 +27,8 @@ def self.welcome_sign_in_message
   def self.continue_btn
     '.interaction-click-control-continue'
   end
-end
\ No newline at end of file
+
+  def self.interactive_verifications_page
+    "Upload document(s) proving your identity"
```
Comment (markgoho) [inline features/support/object_model_pages/your_information.rb:32]: this is still hard-coded english text -- needs to be a css class/attribute or reference a translation

**Review by markgoho — CHANGES_REQUESTED** [review]

File: features/support/object_model_pages/your_information.rb (hunk 2 above)
Comment (harshared) [reply features/support/object_model_pages/your_information.rb:32]: for this we don't have any specific css class/attribute except ul, li. and we don't have translation reference for this

**Comment by harshared** [comment]
check PR: https://github.com/ideacrew/enroll/pull/1366
closing this

**Pull request closed without merging**